charts/app-config/values-integration.yaml

```diff
@@ -1216,8 +1216,6 @@ govukApplications:
             ]}}]
         hosts:
           - name: chat.{{ .Values.k8sExternalDomainSuffix }}
-      uploadAssets:
-        enabled: false
```

Comment:
I think we'll need this to be:

```
uploadAssets:
  path: /app/public/assets/chat
```

so it matches: https://github.com/alphagov/govuk-chat/pull/169/files#diff-c1fd91cb1911a0512578b99f657554526f3e1421decdb9e908712beab57e10f9R44

Reply:
Done 👍